lightkurve/utils.py

```diff
@@ -484,6 +486,16 @@ def detect_filetype(header):
     try:
         # use `telescop` keyword to determine mission
         # and `creator` to determine tpf or lc
+
+        if "SOURCE" in header.keys():
+            # Could be a custom lightcurve
+            if header['SOURCE'] == 'lightkurve interact':
+                if header['mission'].lower() == 'tess':
+                    return 'TessLightCurveFile-interact'
+                elif header['mission'].lower()[0] == 'k':
+                    return 'KeplerLightCurveFile-interact'
```

Comment:
I'm hesitant to define interact-generated files as a different file type just because the flux column has a different name (or are there other key differences?).  I think we should think of them as standard files -- perhaps the only difference being that if you call `lcf.SAP_FLUX` you get a nice error message that says: `This file does not have a SAP_FLUX column. The available flux columns are: ...`